redirects.yaml

```diff
@@ -268,6 +235,43 @@ developers-meeting:
     en: https://contao.org/en/news/recap-of-the-first-contao-core-developers-meeting-2023
     de: https://contao.org/de/news/rueckblick-auf-das-erste-core-entwicklertreffen-2023
 
+# Docs
+docs/manager-password:
+    en: https://docs.contao.org/manual/en/installation/contao-manager/#did-you-forget-your-contao-manager-login-information
+    de: https://docs.contao.org/manual/de/installation/contao-manager/#hast-du-die-zugangsdaten-vom-contao-manager-vergessen
+
+docs/shortcuts:
+    en: https://docs.contao.org/manual/en/administration-area/back-end-keyboard-shortcuts/
+    de: https://docs.contao.org/manual/de/administrationsbereich/backend-tastaturkuerzel/
+
+docs/webroot:
+    en: https://docs.contao.org/manual/en/installation/install-contao/#hosting-configuration
+    de: https://docs.contao.org/manual/de/installation/contao-installieren/#hosting-konfiguration
+
+docs/core-extensions:
+    en: https://docs.contao.org/manual/en/core-extensions/
+    de: https://docs.contao.org/manual/de/core-erweiterungen/
+
+docs/crawler:
+    en: https://docs.contao.org/manual/en/system/maintenance/
+    de: https://docs.contao.org/manual/de/system/systemwartung/
+
+docs/system-requirements:
+    en: https://docs.contao.org/manual/en/installation/system-requirements/
+    de: https://docs.contao.org/manual/de/installation/systemvoraussetzungen/
+
+docs/faq:
+    en: https://docs.contao.org/manual/de/faq/
+    de: https://docs.contao.org/manual/en/faq/
+
+docs/mumble:
+    en: https://docs.contao.org/manual/en/guides/mumble/
+    de: https://docs.contao.org/manual/de/anleitungen/mumble/
+
```

Comment:
```suggestion
```

I don‘t think we need this anymore.

Reply:
Yeah I thought so too but it's still in the docs and I think we should remove them in a different PR. It's unrelated.

Reply:
Please note that this only removes the **new** link, not the existing one. We have to keep the latter of course, but we do not need to create a new one.

Reply:
We can, but I cannot take your suggestion then because it would break the existing link. I've updated the PR.